<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profortosi_new_1_bClass</class>
 <widget class="QDialog" name="profortosi_new_1_bClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Νέα Προφόρτωση</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Φορτηγό</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCar2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushCode">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>2.Κατά κωδικό</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Πελάτης</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBack">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushLoop">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>1.Κατά συρροή</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCar1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelCustomer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
